framework/src/modules/nft/events/all_nfts_from_chain_suported.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseEvent, EventQueuer } from '../../base_event';
+import { LENGTH_CHAIN_ID } from '../constants';
+
+export interface AllNFTsFromChainSupportedEventData {
+	chainID: Buffer;
+}
+
+export const supportAllNFTsFromChainEventDataSchema = {
```

Comment:
I think names should be consistent for file, interface and schema. In this case schema name is different from file and interface. same for other events where applicable.